lxd/api_internal.go

```diff
@@ -1011,7 +1012,8 @@ func internalImport(d *Daemon, r *http.Request) Response {
 			BaseImage:    baseImage,
 			Config:       snap.Config,
 			CreationDate: snap.CreatedAt,
-			Ctype:        db.CTypeSnapshot,
+			Type:         instance.TypeContainer,
+			IsSnapshot:   true,
```

Comment:
Probably ought to be just `Snapshot` to line up with `Stateful`.
You should also look at everywhere we fill a db.ContainerArgs struct to make sure that you don't forget to set it accordingly as we do have many of those and have gotten bit in the past by forgetting to copy/set a new attribute on that struct.